ArduCopter/mode_auto.cpp

```diff
@@ -992,6 +994,8 @@ bool ModeAuto::verify_command(const AP_Mission::Mission_Command& cmd)
     // do commands (always return true)
     case MAV_CMD_DO_CHANGE_SPEED:
     case MAV_CMD_DO_SET_HOME:
+    case MAV_CMD_DO_SET_ROI_LOCATION:
+    case MAV_CMD_DO_SET_ROI_NONE:
```

Comment:
Is there any reason not to do this in `AP_Mission::verify_command()`?

Reply:
Hi @peterbarker, thanks for the review.  Functionally that would be fine but I think it would be odd to have half the implementation in AP_Mission and half in the vehicle code.